conf/master

```diff
@@ -303,11 +309,13 @@
 # of 8 will be used. If no batch_safe_limit is specified, then no automatic
 # batching will occur.
 #batch_safe_limit: 100
+#
 #batch_safe_size: 8
 
 # Master stats enables stats events to be fired from the master at close
 # to the defined interval
 #master_stats: False
+
```

Comment:
don't need the extra line

Reply:
`False` **is** the default so *should* be followed by a blank line.